### ISSUE TITLE
Milestone creation error

### ISSUE THREAD

**Issue by quangbahoa** [Leantime/leantime#108]
Got some errors white create new milestone

```
snap.svg-min.js:19 Error: <rect> attribute width: A negative value is not valid. ("-695857.5833333334")
d @ snap.svg-min.js:19
snap.svg-min.js:19 Error: <rect> attribute width: A negative value is not valid. ("-73248.16666666666")
d @ snap.svg-min.js:19
(index):5 Uncaught SyntaxError: Invalid or unexpected token
    at eval (<anonymous>)
    at jquery-1.9.1.min.js:3
    at Function.globalEval (jquery-1.9.1.min.js:3)
    at e.fn.init.domManip (jquery-1.9.1.min.js:4)
    at e.fn.init.append (jquery-1.9.1.min.js:4)
    at Object._writeContent (jquery.nyroModal.custom.js:502)
    at Object.<anonymous> (jquery.nyroModal.custom.js:447)
    at i (jquery-1.9.1.min.js:3)
    at Object._contentLoading (jquery.nyroModal.custom.js:460)
    at Object._setCont (jquery.nyroModal.custom.js:328)
```

LT 2.0.8
PHP 7.4
Mariadb 10.2
Nginx 1.17

**Comment by marcelfolaron** [comment]
Is that happening every time you create a milestone? This looks like the end date was before the start date (which shouldn't be possible). Can you tell me what you filled into the form?

**Comment by quangbahoa** [comment]
> Is that happening every time you create a milestone? This looks like the end date was before the start date (which shouldn't be possible). Can you tell me what you filled into the form?

Yea, I leave end date blank.

**Comment by marcelfolaron** [comment]
I see. Will get a fix out as soon as possible

**Comment by quangbahoa** [comment]
I think we should allow blank end date as infinite deadline for project.

**Comment by marcelfolaron** [comment]
I agree, that makes sense for projects but I believe Milestone s (by
definition) should have an end date.

I am working on a Kanban and Gantt visualization for Projects as well. So
that would apply there.

**Comment by marcelfolaron** [comment]
Fixed in V2.0.10
